kernel/arch/dreamcast/hardware/cdrom.c

```diff
@@ -371,21 +374,21 @@ int cdrom_cdda_play(uint32 start, uint32 end, uint32 repeat, int mode) {
 }
 
 /* Pause CDDA audio playback */
-int cdrom_cdda_pause(void) {
+enum cd_cmd_response cdrom_cdda_pause(void) {
     int rv;
```

Comment:
In this case, the variable "rv" can be made of this type.
I would also add typedef for enums to avoid using the word "enum" everywhere.